rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,233 @@
+   </td>
+  </tr>
+  <tr>
+   <td>Flowcell ID
+   </td>
+   <td>The Illumina identifier of the flowcell id
+   </td>
+   <td rowspan="3" >The unique combination of Flowcell ID, Lane and Barcode uniquely identifies any multiplexed library on an Illumina Flowcell
+   </td>
+  </tr>
+  <tr>
+   <td>Lane in Flowcell
+   </td>
+   <td>The lane in the flowcell
+   </td>
+  </tr>
+  <tr>
+   <td>Barcode of Library
+   </td>
+   <td>Barcode (or concatenation of the barcodes) of the library
+   </td>
+  </tr>
+</table>
+
+## Metadata Requirements for SS2 scRNA-seq datasets and provision for plate-based analyses
+The above metadata requirements are also compatible with SS2 analyses and can support plate based-analyses. Cells that have been sequenced more than once (a rare, but plausible scenario for SS2) can be identified by using a library identifier as presented in this RFC, currently under review.
+
+Furthermore, this approach can be extended to support plate-based processing. By ensuring that a plate field is provided for each cell, multiple cell bundles can be aggregated into a single plate-based workflow. A DAPS can contain one or more plates. Provision must be made for any cells not associated with a plate identifier, or otherwise, the plate identifier must be a compulsory field.
+
+## Pros and Cons of DAPS Approach
+
+Pros
+*   Avoids potentially very expensive run-away queries
+*   Ensures that analyses are triggered once with the appropriate datasets
+*   Is flexible and can accommodate all foreseeable data modalities
+*   Can immediately resolve processing challenges with 10X datasets
+*   Can provide a mechanism for moving to plate-based processing for SS2 in a seamless manner
+*   Does not preclude future automation of pipeline initiation when a mature system state is achieved
+*   Re-uses existing notification infrastructure
+*   Is permanently saved in the DSS and analyses are reproducible
+*   Provides a natural place to provide analysis-specific parameters
+
+Cons
+*   Analysis need to modify pipeline initiation scripts
+*   Initial implementation has human time cost of triggering the pipeline runs
+*   Requires the human who is triggering the pipeline to know what criteria need to be satisfied to trigger (i.e. the project is complete)
+
+## Alternative Approaches 
+
+The following section summarizes possible alternative approaches for ensuring correct processing, along with the pros and cons of each. These approaches are not recommended by this RFC but are listed for comparison purposes.
+
+*   Placing all the data that needs to be co-processed in a single bundle and continue to use bundle notifications
+    *   Pros
+        *   Essentially no extra work from infrastructure required
+    *   Cons
+        *   Wranglers are required to package data in such a way to satisfy the requirements of the analysis, the details of which they should not be aware
```

Comment:
Why the desire to avoid wranglers understanding what makes reasonable input for an analysis process? Surely they need to understand this to ensure they are having the correct conversations with submitters to get the right information in the first place? It isn't clear why that understanding is a bad thing, certainly at this stage in the project

Reply:
There is no desire for wrangler avoiding understanding anything.  there is a requirement that wranglers do not have to package the data based on analysis pipelines that may not even exist when the data is ingested.  This needs to be made clearer